pkg/model/file/interfaces.go

```diff
@@ -40,7 +40,7 @@ type Template interface {
 	Builder
 	// GetBody returns the template body
 	GetBody() string
-	// SetTemplateDefaults returns the TemplateMixin for creating a scaffold file
+	// SetTemplateDefaults sets the default values for templates
```

Comment:
Nit which we are tryng to address here: 

<img width="833" alt="Screenshot 2020-02-25 at 12 57 11" src="https://user-images.githubusercontent.com/7708031/75249522-6229af80-57ce-11ea-913f-88193df5f370.png">

However, I could not find any implementation of `SetTemplateDefaults` which is returning an error actually. So, hould not bee better we change it  as follows:

```suggestion
	// SetTemplateDefaults sets the default values for templates
	SetTemplateDefaults() 
```

Reply:
I could find at least one case when it returns an error: `pkg/scaffold/internal/templates/v1/gopkg.go`. Skipping the rest of v1 files, will check v2 files now.

Reply:
Found one file in v2 too: `pkg/scaffold/internal/templates/v2/kustomize.go`